app/metrics.yaml

```diff
@@ -509,6 +509,220 @@ metrics:
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  search_bookmarks:
+    type: string_list
+    description: >
+      Whether or not the user has enabled bookmark search suggestions
+      default: true
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  search_browsing_history:
+    type: string_list
+    description: >
+      Whether or not the user has enabled browsing history suggestions.
+      default: true
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  show_clipboard_suggestions:
+    type: string_list
+    description: >
+      Whether or not the user has enabled clipboard search suggestions.
+      default: true
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  show_search_shortcuts:
+    type: string_list
+    description: >
+      Whether or not the user has enabled search shortcuts.
+      default: true
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  open_links_in_a_private_tab:
+    type: string_list
+    description: >
+      Whether or not the user has enabled open links in a private tab.
+      default: false
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  sync:
+    type: string_list
+    description: >
+      Whether or not the user is signed into FxA
+      default: false
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  sync_items:
+    type: string_list
+    description: >
+      The list of items the user has chosen to sync with FxA.
+      default: "" if the user is signed out. Otherwise defaults to:
+      [bookmarks, history, passwords, tabs]
```

Comment:
Don't we default to what the user has enabled in their Firefox Account? And if they don't have one and are creating it with Fenix, it's the defaults you listed.

Reply:
Ah, good call out! Thanks 😄